### PR TITLE
Created config file for Certification Hours (Issue #39284)

### LINKED ISSUE #39284: Typo in scientific computing with python certificate
**Describe your problem and how to reproduce it:**
There is a typo in the scientific computing with python certificate which displays "400" hours instead of "300" as the conservative estimated time of completion.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/certification/obsessedyouth/scientific-computing-with-python-v7

**Tell us about your browser and operating system:**
* Browser Name: 
Google Chrome
* Browser Version: 
Version 84.0.4147.89 (Official Build) (64-bit
* Operating System: 
Windows 10 Enterprise

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
Screenshot (364)

### PR DESCRIPTION
**Pull request by HiThereImCam**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39284 

Added every certification to my certHours object in completionHours-settings.js so that just in case FCC wants to change the hours for some certs in the future, it's easily adjustable.

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
Thanks for the pull request.
Could you please add some api and client tests.

**Comment by HiThereImCam** [comment]
@ahmadabdolsaheb Thanks for the reply. I'll be right on it. Do I have to close this PR and create a new PR with requests?

**Comment by ahmadabdolsaheb** [comment]
@HiThereImCam, let me know if you need help with the tests.

**Comment by HiThereImCam** [comment]
@ahmadabdolsaheb Currently on gitter trying to resolve testing issues. Thank you for your help, The tests themselves are straight forward. Getting jest to run the test file is proving to be troublesome.

**Comment by HiThereImCam** [comment]
@ahmadabdolsaheb Hi Ahmad, I was able to

**Comment by ahmadabdolsaheb** [comment]
Also it seems like you are making a pr from your master branch.
Could you create another branch with a descriptive name and make a pr accordingly.

**Comment by HiThereImCam** [comment]
@ahmadabdolsaheb yes absolutely. The guys on gitter also pointed that out. Just to give a status update, I'll be pushing a new PR over the weekend. Thanks!

File: config/completionHours-settings.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const certHours = {
```
Comment (KartikSoneji) [inline config/completionHours-settings.js:1]: Please use the same style as before.

```javascript
//might need to fix the path
import certTypes from '../api-server/server/utils/certTypes.json';

const certHours = {
  [certTypes.frontEnd]: 400,
  [certTypes.backEnd]: 400,
  [certTypes.dataVis]: 400,
//...
}
```

File: api-server/server/boot/certificate.js [hunk 2]
```diff
@@ -179,24 +179,6 @@ const certText = {
   [certTypes.machineLearningPyV7]: 'Machine Learning with Python'
 };
 
-const completionHours = {
```
[the remark this answers is not in the document]
Comment (HiThereImCam) [reply api-server/server/boot/certificate.js:182]: @KartikSoneji thank you so much for the feedback.

**Comment by ahmadabdolsaheb** [comment]
Hi @HiThereImCam, how is the progress on this PR? We are looking forward to have this change in production. let me know if you need any help.

**Comment by HiThereImCam** [comment]
@ahmadabdolsaheb Hey ahmad! Thanks for checking up on me. Sorry, I got busy(went back to school) but I do have time to work on this over the weekend. If I have any questions, I'll let you know. Thanks!

**Comment by ahmadabdolsaheb** [comment]
@HiThereImCam, how is the progress on this PR?

**Pull request closed without merging**